libraries/AP_HAL_ChibiOS/hwdef/JHEM_JHEF405/hwdef-bl.dat

```diff
@@ -0,0 +1,43 @@
+
+# hw definition file for processing by chibios_hwdef.py
+# for JHEF405 hardware.
+# thanks to betaflight for pin information
+
+# MCU class and specific type
+MCU STM32F4xx STM32F405xx
+
+# board ID for firmware load
+APJ_BOARD_ID 1081
```

Comment:
can use AP_HW_JHEMCUGF16F405 instead of the numebr

Reply:
fixed